bots/naughty/rhel-7/9641-xtermjs-dimensions-crash

```diff
@@ -1,4 +1,4 @@
 Traceback (most recent call last):
-  File "test/verify/check-machines", line *, in testSerialConsole
+  File "test/verify/machineslib.py", line *, in testSerialConsole
```

Comment:
Please squash that commit into the one that splits the test case. Also, it's "naughty", not "naughtly" :)

Reply:
Done